misp_stix_converter/converters/buildMISPAttribute.py

```diff
@@ -23,55 +23,170 @@
 ipre = re.compile("([0-9]{1,3}.){3}[0-9]{1,3}")
 log = logging.getLogger("__main__")
 
+# Dedicated to parse related object
+# Support for first level "objRel.related_objects"
 def parseRelated(obj, mispEvent, pkg):
-    for i in obj:
-
-        if i.properties.file_name:
-            mispEvent.add_attribute('filename', six.text_type(i.properties.file_name), comment=pkg.title or None)
-
-        if i.properties.size_in_bytes:
-            mispEvent.add_attribute('size-in-bytes', six.text_type(i.properties.size_in_bytes), comment=pkg.title or None)
- 
-        if i.properties.md5:
-            if len(i.properties.md5) == 32:
-                mispEvent.add_attribute('email-attachment', six.text_type(i.properties.md5), comment=pkg.title or None)
     
-        if i.properties.sha1:
-            if len(i.properties.sha1) == 40:
-                mispEvent.add_attribute('email-attachment', six.text_type(i.properties.sha1), comment=pkg.title or None)
-
-        if i.properties.sha256:
-            if len(i.properties.sha256) == 64:
-                mispEvent.add_attribute('email-attachment', six.text_type(i.properties.sha256), comment=pkg.title or None)
+    for i in obj.related_object:
+        type_ = type(i.properties)
+        # Here comes the sun (and of course the fun) for related objects! (DB)
+        if type_ == address_object.Address:
+            buildAddressAttribute(i.properties, mispEvent, pkg)
+                    
+        elif type_ == domain_name_object.DomainName:
+            buildDomainNameAttribute(i.properties, mispEvent, pkg)
+                    
+        elif type_ == hostname_object.Hostname:
+            buildHostnameAttribute(i.properties, mispEvent, pkg)
+                    
+        elif type_ == uri_object.URI:
+            buildURIAttribute(i.properties, mispEvent, pkg)
+                
+        elif type_ == file_object.File:
+            buildFileAttribute(i.properties, mispEvent, pkg)
 
+        elif type_ == email_message_object.EmailMessage:
+            buildEmailMessageAttribute(i.properties, mispEvent, pkg)
+        else:
```

Comment:
No need for the else/pass - if the type isn't caught by the `elif`s above, it'll pass anyhow